build_tools/waf_dynamo.py

```diff
@@ -1665,14 +1666,17 @@ def detect(conf):
     except BuildUtilityException as ex:
         conf.fatal(ex.msg)
 
+    target_os = build_util.get_target_os()
+    conf.env['TARGET_OS'] = target_os
```

Comment:
Store target OS into environment to easier retrieve it later in every module's wscript